memberships/views.py

```diff
@@ -156,8 +156,8 @@ def confirm(request):
     if not request.user.is_authenticated:
         return HttpResponseRedirect(reverse("register"))
 
-    donation = request.GET.get("donation")
-    total = 1 if not donation else int(donation) + 1
+    donation = float(request.GET.get("donation"))
```

Comment:
@Freshrojek I am not around to fix it but the errors are from this, when there is no donation, it cannot get the float value.

Reply:
Nice, will jump on it today some time 👍🏼